xinference/model/llm/llm_family.py

```diff
@@ -711,7 +714,9 @@ def cache_from_csghub(
                 },
                 llm_spec.model_id,
                 file_name=filename,
-                endpoint="https://hub-stg.opencsg.com",
+                endpoint=os.environ.get(
+                    XINFERENCE_ENV_CSG_ENDPOINT, "https://hub-stg.opencsg.com/"
```

Comment:
Since the endpoint is used twice, we can add an variable to `constants.py`, for instance, `CSG_ENDPOINT`.